src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2278,6 +2275,23 @@ public void testPreSignedUrlHeaderOverrides() throws Exception {
         Assert.assertEquals(contentDisposition, response.getHeaders().getFirst(RestUtil.HEADER_CONTENT_DISPOSITION));
     }
 
+    @Test
+    public void testVPoolHeader() {
+        CreateBucketRequest createBucketRequest = new CreateBucketRequest("looney-bucket")
+                .withVPoolId("plylab-sp1");
+        client.createBucket(createBucketRequest);
+    }
+
+    @Test
```

Comment:
add a comment here that states you must use a debug proxy, like Fiddler or Charles, in order to verify the custom header is present.. otherwise, we could inject our own Jersey filter to sniff for it

Reply:
✔️